### PR TITLE
Add "create repository project" and "create project column" methods

### PR DESCRIPTION
**Pull request by blatinier**
Added the two methods:
 - Create a project for a given repo
 - Create a column for a given project 

Tests are basics but done :)

### CODE REVIEW

**Comment by blatinier** [comment]
Hum I don't really understand how I should handle failing tests. Did I messed up the recorded file, or do I need to fix all repos with the new "/blatinier"?

File: github/MainClass.py [hunk 1]
```diff
@@ -775,43 +775,37 @@ def get_access_token(self, installation_id, user_id=None):
...
-            headers={
-                "Authorization": "Bearer {}".format(self.create_jwt()),
-                "Accept": Consts.mediaTypeIntegrationPreview,
-                "User-Agent": "PyGithub/Python"
-            },
-            body=body
-        )
-        response = conn.getresponse()
-        response_text = response.read()
+        response = requests.post("https://api.github.com/installations/{}/access_tokens".format(installation_id),
```
Comment (sfdye) [inline github/MainClass.py]: Why did you change this file?

Comment (sfdye) [reply github/MainClass.py]: If you really want to change this, I would like to see them in a separate PR please

**Comment by sfdye** [comment]
> Did I messed up the recorded file

I think so. Things become a little bit tricky when you work on some else's PR. You may want to revert `Project.setUp.txt` to begin with. Setting up the tests locally may help.

File: github/MainClass.py (hunk 1 above)
Comment (blatinier) [reply github/MainClass.py]: Opened a new PR #998 for that. I will update this one.
It's good for codebase coherence I think.
Plus official doc says: 

> See also: The Requests package is recommended for a higher-level HTTP client interface. 

I'll update this PR.

**Comment by blatinier** [comment]
Ok I managed to fix tests locally.

File: github/MainClass.py (hunk 1 above)
Comment (sfdye) [reply github/MainClass.py]: Awesome, will review
